software/software-news.rst

```diff
@@ -26,6 +26,13 @@ As a result of the upgrade, the following packages are now default:
     "GNU Compiler Collection (GCC)", "12.2.0", "12.3.0"
     "AMD Compiler (AMD)", "5.3.0", "5.7.1"
 
+In addition to the default modules changing, the layout of the modules for the OLCF provided software stack will change.
+Going forward our core software (i.e software independent of compiler type/version) will be in a default module
+called `Core` with the version being in the format `year/month`. Additionally the suffixes to modules have
```

Comment:
Maybe instead of "Additionally the suffixes to modules have been reworked", we can say "Explicit suffixes have been added to modules to indicate MPI, OpenMP, and GPU support." Technically, we had very few suffixes before, so we didn't rework too much, right?

Reply:
I agree. Should be updated now.